shared/data/fingerprint-protection.js

```diff
@@ -291,6 +282,32 @@
         return windowScript
     }
 
+    /**
+     * Temporary storage can be used to determine hard disk usage and size.
+     * This will limit the max storage to 4GB without completely disabling the
+     * feature.
+     */
+    function modifyTemporaryStorage () {
+        const randomFunctionName = 'A' + Math.random().toString(36).substring(2) // random string of alphanumeric chars
+        const script = `
+            if (navigator.webkitTemporaryStorage) {
+                try {
+                    ${randomFunctionName} = navigator.webkitTemporaryStorage.queryUsageAndQuota
```

Comment:
I'm still unsure why we need this random name 🤔 If we did:

```suggestion
                    const org = navigator.webkitTemporaryStorage.queryUsageAndQuota
```

no one, outside of the try/catch block, would have access to that `org` variable and we could still use it inside `queryUsageAndQuota`.